addons/website_forum/static/src/js/website_forum.js

```diff
@@ -494,6 +503,62 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         $('.forum_intro').slideUp();
         return true;
     },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onClickFlagValidator: function (ev) {
+        ev.preventDefault();
+        const $currentTarget = $(ev.currentTarget);
+        const data = $currentTarget.data();
+        this._rpc({
+            model: 'forum.post',
+            method: data.action,
+            args: [data.postId],
+        }).then(result => {
+            if (result) {
+                let $flaggedButton = $currentTarget.parent().toggleClass('d-none').siblings('button'),
+                    $child = $flaggedButton.children(),
+                    $count_flagged_posts = $('span#count_flagged_posts'),
```

Comment:
```suggestion
                    $countFlaggedPosts = $('span#count_flagged_posts'),
```
As per guidelines. Linter should tell you that.
![image](https://user-images.githubusercontent.com/30048408/124309632-52ad3a00-db6b-11eb-8ac5-c2ddc293bf5b.png)